Jenkinsfile

```diff
@@ -39,12 +39,12 @@ pipeline {
   parameters {
     string(
       name: 'TOOLS_VERSION',
-      defaultValue: '15.3.0',
+      defaultValue: '-j -b markp_xsim_expose_signals_from_usb_shim latest',
```

Comment:
This may be valid but I don't understand it!

Reply:
It's grabbing a build of the tools from a Jenkins run